tests/admin_inlines/tests.py

```diff
@@ -164,10 +164,10 @@ def test_tabular_non_field_errors(self):
         }
         response = self.client.post(reverse('admin:admin_inlines_titlecollection_add'), data)
         # Here colspan is "4": two fields (title1 and title2), one hidden field and the delete checkbox.
-        self.assertContains(
-            response,
+        self.assertInHTML(
             '<tr class="row-form-errors"><td colspan="4"><ul class="errorlist nonfield">'
-            '<li>The two titles must be the same</li></ul></td></tr>'
+            '<li>The two titles must be the same</li></ul></td></tr>',
+            response.content.decode(),
```

Comment:
I'm not sure why this change is necessary :thinking:

Reply:
I think it was due to whitespace changes, but I think the templates have evolved so we don't need this change anymore. I'll revert this change.